src/doc/rustdoc/src/unstable-features.md

```diff
@@ -389,3 +389,16 @@ pub struct BigX;
 
 Then, when looking for it through the `rustdoc` search, if you enter "x" or
 "big", search will show the `BigX` struct first.
+
+### `index-page` feature
```

Comment:
Since these are CLI flags, they should follow the convention of the other flags' headings:

```markdown
### `--index-page`: provide a top-level landing page for docs

### `--enable-index-page`: generate a default index page for docs
```

(The section about `doc_alias` is also in the wrong place, but i failed to catch that when you wrote it in the first place... `>_>`)

Reply:
Ah. I'll fix all that at once then.